src/Opdex.Platform.WebApi/Models/Requests/Transactions/TransactionBroadcastNotificationRequest.cs

```diff
@@ -12,4 +12,6 @@ public class TransactionBroadcastNotificationRequest
     /// </summary>
     /// <example>402aa2241adb7b04d07d4dbc89f8aae72fa1c11f9bd2bd9013222cd774ed39fe</example>
     public Sha256 TransactionHash { get; set; }
+
+    public Address PublicKey { get; set; }
```

Comment:
This needs documenting in openapi.yml

nitpick: Consider renaming to Address, as I think public key is a more ambiguous term.

Reply:
Renaming would require changing SSAS spec